lxd/cgroup/init.go

```diff
@@ -6,20 +6,24 @@ import (
 	"path/filepath"
 	"strings"
 
+	lxc "gopkg.in/lxc/go-lxc.v2"
```

Comment:
I want to keep this package free of `go-lxc`, can you instead have the check done in the ReadWriter we create in `container_lxc.go`?

Reply:
Oh right, I forgot about that constraint... Let me see...